manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "10up Scaffold",
+  "short_name": "10up",
+  "description": "Project description.",
+  "display": "standalone",
+  "start_url": "/?utm_source=app",
+  "background_color": "#232323",
+  "orientation": "portrait",
```

Comment:
Let's remove this or update to, "any" we don't want to force an orientation: https://developers.google.com/web/fundamentals/web-app-manifest/#orientation